feature-libs/product-configurator/rulebased/components/price/configurator-price.component.html

```diff
@@ -17,3 +17,8 @@
     <div class="cx-price-total">{{ priceTotal }}</div>
   </ng-container>
 </ng-container>
+<ng-container *ngIf="displaySelected()">
+  <div [ngClass]="styleClass">
```

Comment:
Breaking, needs to be handled

Reply:
Do we want to downport this fix to an previous versions?